packages/core/src/external/aws/lambda-logic/document-bulk-signing.ts

```diff
@@ -0,0 +1,24 @@
+import { S3Utils } from "../s3";
+
+const SIGNED_URL_DURATION_SECONDS = 3600; // longer since a lot of docs
```

Comment:
is this too long? We want it to be longer than 60 seconds since the customer will not get each signed url immediately, and is instead getting them in bulk

Reply:
I think we wanted the timeout to use the signed URL to be 5min, no?